src/internal/text/index.ts

```diff
@@ -2,5 +2,4 @@ export { TextBody } from './TextBody';
 export { TextHeadline } from './TextHeadline';
 export { TextLabel1 } from './TextLabel1';
 export { TextLabel2 } from './TextLabel2';
-export { TextMessage } from './TextMessage';
```

Comment:
hehehe yeah I agree, I wasn't quite sure to have it. Was more a quick decion.

Thank you for correcting this.